x-pack/plugins/apm/server/routes/services/get_services/get_services_from_error_and_metric_documents.ts

```diff
@@ -59,9 +61,7 @@ export async function getServicesFromErrorAndMetricDocuments({
         },
         aggs: {
           sample: {
-            random_sampler: {
-              probability,
-            },
+            random_sampler: { probability, ...(seed ? { seed } : {}) },
```

Comment:
It would be nice if `seed` and `probability` were colocated in a randomSampler object that can be passed around easier and with more safety:

```ts
const randomSampler = { seed, probability };
```

```suggestion
            random_sampler: randomSampler,
```

Reply:
Done in https://github.com/elastic/kibana/pull/133789/commits/49fd905f697cc82971b4d2b7906d4246fa090a27